scalardl-web-client-sdk.js

```diff
@@ -19,18 +19,18 @@ class ClientService extends ClientServiceBase {
    * @param {Object} properties JSON Object used for setting client properties
    */
   constructor(properties) {
-    const host = properties['scalar.ledger.client.server_host'];
-    const tlsEnabled = properties['scalar.ledger.client.tls.enabled'];
+    const host = properties['scalar.dl.client.server.host'];
+    const tlsEnabled = properties['scalar.dl.client.tls.enabled'];
 
     const ledgerClientServiceURL =
-      `${tlsEnabled ? 'https' : 'http'}://${host}:${properties['scalar.ledger.client.server_port']}`;
+      `${tlsEnabled ? 'https' : 'http'}://${host}:${properties['scalar.dl.client.server.port']}`;
     const ledgerPriviledgedClientServiceURL =
-      `${tlsEnabled ? 'https' : 'http'}://${host}:${properties['scalar.ledger.client.server_privileged_port']}`;
+      `${tlsEnabled ? 'https' : 'http'}://${host}:${properties['scalar.dl.client.server.privileged_port']}`;
 
     const services = {
       ledgerClient: new LedgerClient(ledgerClientServiceURL),
       ledgerPrivileged: new LedgerPrivilegedClient(
-          ledgerPriviledgedClientServiceURL
+          ledgerPriviledgedClientServiceURL,
```

Comment:
Do we need this ?

Reply:
This is made by ESLint (using Google JavaScript Style Guide).
However, I checked the Google JavaScript Style Guide, trailing comma is applied in arrays and objects but not function parameters.
Would you like to revise it back?

Reply:
It is fine. I was just wondering. Thank you !